.github/ISSUE_TEMPLATE/40_tooling.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Tool idea
+about: I have an idea for, or need, specific tooling for CFF
```

Comment:
I would prefer to not mention CFF here, because that refers more to the format which is better covered in the other issue template. Maybe we can refer to `CITATION.cff` files instead? That would probably cover any tooling needs, right?

Reply:
Done in 4e9c5eef026264fbe8cae7c49e82f3f148aa0893.